src/pages/tezos/index.js

```diff
@@ -76,14 +77,58 @@ const TestimonialCard = ({ link, img, name, username, text }) => {
 };
 
 function Landing() {
+  const [videoModal, setVideoModal] = useState(0);
+  function VideoModal() {
+    return (
+      <div
+        className={`bg-base-700 px-12 py-16 rounded-lg relative flex flex-col items-center shadow-lg text-center`}
+        style={{ maxWidth: '60vw' }}
+      >
+        <div
+          onClick={() => setVideoModal(0)}
+          className="h-12 w-12 rounded-full bg-base-500 absolute right-8 top-8 cursor-pointer flex items-center justify-center"
+        >
+          <MdClose size="24px" />
+        </div>
+        <div>
+          <iframe
+            width="800"
+            height="450"
+            src="https://www.youtube.com/embed/8VrlUL8W__Q?autoplay=1"
```

Comment:
This is the link to phase 1 of the project. Needs to be replaced with the phase 2 video.